### PR TITLE
Fix config, buttons and pages styling

### PR DESCRIPTION
**Pull request by sebastien-comeau**
### Description

List of proposed changes:

- Fix tailwing css head rem unit
- Fix action and link buttons styling
- Fix index page styling
- Fix landing page styling
- Fix expectation page styling
- Fix input components styling
- Fix lists styling

### What to test for/How to test

Load the application and browse the application

### Additional Notes

ActionButton and LinkButton component logic changed based on TailwindCSS documatation:
https://tailwindcss.com/docs/content-configuration#:~:text=By%20using%20this%20very%20simple%20approach%2C%20Tailwind%20works%20extremely%20reliably%20with%20any%20programming%20language%2C%20like%20JSX%20for%20example%3A

GCWeb button style
https://design.canada.ca/common-design-patterns/buttons.html

### CODE REVIEW

**Review by krischarbonneau — COMMENTED** [review]
Looks fantastic. I like the changes done to the buttons since they didn't look that great on smaller resolutions before. One minor comment but aside from that everything else looks good to me.

File: tailwind.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     extend: {
       fontFamily: {
         display: ['Lato', 'sans-serif'],
-        body: ['Noto sans', 'sans-serif'],
+        body: ['"Noto sans"', 'sans-serif'],
```
Comment (krischarbonneau) [inline tailwind.config.js:10]: Extra quotes here

Comment (sebastien-comeau) [reply tailwind.config.js:10]: I'm used to always add double quotes to font-family that have multiple words.

Example from Taillwind base css:
image

Comment (sebastien-comeau) [reply tailwind.config.js:10]: Note: If a font name contains white-space, it must be quoted.
https://www.w3schools.com/cssref/pr_font_font-family.php#:~:text=If%20a%20font%20name%20contains%20white%2Dspace%2C%20it%20must%20be%20quoted

Comment (krischarbonneau) [reply tailwind.config.js:10]: The more your know 😄 Just checked Tailwind's documentation and they're doing the same thing here: https://tailwindcss.com/docs/font-family#customizing-your-theme so I'm good with this.

**Review by krischarbonneau — APPROVED** [review]

**Pull request merged**